src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,39 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER_OF_ELEMENTS = 10;
+    private K[] keys = (K[]) new Object[MAX_NUMBER_OF_ELEMENTS];
+    private V[] values = (V[]) new Object[MAX_NUMBER_OF_ELEMENTS];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
zero is the default value for int type